doc/website/_data/navigation.yml

```diff
@@ -0,0 +1,62 @@
+# main links
+main:
+  - title: "Get Started"
+    url: /get-started/overview/
+  - title: "Docs"
+    url: /docs/sample-example/
+  - title: "API"
+    url: "https://quekyj.github.io/rohd/api"
+
+# get started links
+get-started:
```

Comment:
We'll probably want to adjust the original README so that it's not so long and instead links to some of this documentation

Reply:
Ya, good idea. I am thinking that Readme should be shorter and just have links. Maybe something like this is good.
https://github.com/tensorflow/tensorflow

Reply:
As part of this PR, let's wipe out what's in README down to just links to the website.  This PR should get us to the point where we can merge it and the website is ready to be used immediately, then we can work on improving the website from there